apps/ex_wire/lib/ex_wire/packet/get_block_headers.ex

```diff
@@ -11,15 +11,22 @@ defmodule ExWire.Packet.GetBlockHeaders do
   ```
   """
 
+  alias Blockchain.Block
+  alias ExWire.Bridge.Sync
   alias ExWire.Packet
   alias ExWire.Packet.BlockHeaders
+  require Logger
 
   @behaviour ExWire.Packet
 
+  @sync Application.get_env(:ex_wire, :sync_mock, Sync)
+  # TODO: What's a reasonable number for this?
```

Comment:
Ayrat once pointed out it might be better to start opening issues then leaving TODO's

Reply:
Was just for discussion in the PR. I'll remove it before merging. I'll look into what other clients do, but if nothing else, I'll just put in a number, and we'll change it if/when it becomes an issue.